mne/viz/_3d.py

```diff
@@ -464,6 +478,78 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
                                'number of elements as PySurfer plots that '
                                'will be created (%s)' % n_split * n_views)
 
+    # Set default transparency if user didn't specify it
+    if transparent is None:
+        if colormap == 'mne_analyze':
+            transparent = False
+        else:
+            transparent = True
+
+    scale_pts = [fmin, fmid, fmax]
+    # Check if using old fmin/fmid/fmax behavior
+    if limits is None:
+        # Throw deprecation warning (should only show up once)
+        warnings.warn('Using fmin, fmid, fmax is deprecated and will be'
+                      ' removed in v0.10. Use "limits" instead.',
+                      DeprecationWarning)
+        for fi, f in enumerate(scale_pts):
+            if f is None:
+                scale_pts[fi] = [5., 10., 15.][fi]
+
+    # Using new cmap limit behavior
+    else:
+        if any([f is not None for f in [fmin, fmid, fmax]]):
+            warnings.warn('"limits" overrides fmin, fmid, fmax')
+
+        # Colormap is 'mne_analyze'. Need to construct cmap and get scale vals
+        if colormap == 'mne_analyze':
+            if limits == 'auto':
+                ctrl_pts = np.percentile(np.abs(stc.data), [97, 98.5, 99.9])
+                colormap = mne_analyze_colormap(ctrl_pts)
```

Comment:
are you sure this is a good choice? does it produce the images you shared above? If we end up with a collection of scattered sparse islands of activation this is maybe not what we want.